cmd/lukso/utils.go

```diff
@@ -42,7 +42,9 @@ func prepareLogfileFlag(logDir, dependencyName string) string {
 	return fmt.Sprintf("--log-file=%s", prysmFullLogPath)
 }
 
-func createJwtSecret(dest string) error {
+// The lint says this function is unused
+// I'll ignore it for now as I'm missing context
```

Comment:
We can't pass any jwt secret to clients - only path pointers. So ideally user shouldn't worry about creating and putting jwt secrets in any weird, unknown places - this function creates said secret, so I think we should leave it as is and integrate when the time is good

Reply:
Thanks for the explanation